clients/merge-go-ethereum/Dockerfile

```diff
@@ -5,8 +5,8 @@
 # needing to constantly rebuild.
 
 FROM alpine:latest
-ARG repo=MariusVanDerWijden/go-ethereum
-ARG branch=merge-kiln-v2
+ARG repo=ethereum/go-ethereum
+ARG branch=master
```

Comment:
Can't we just remove this client now, and use regular go-ethereum?

Reply:
Yes

Reply:
I've raised a separate PR #515  to remove the merge definition (based on this branch).